src/server/req_manager.c

```diff
@@ -1561,17 +1612,27 @@ mgr_sched_set(struct batch_request *preq)
 	int	  bad_attr = 0;
 	svrattrl *plist;
 	int	  rc;
+	pbs_sched *psched;
+
+	psched = find_scheduler(preq->rq_ind.rq_manager.rq_objname);
```

Comment:
In the existing implementation the scheduler has name. It is "1". 

Please take look into IFL call to update the sched object 
pbs_manager(connector,
		MGR_CMD_SET, MGR_OBJ_SCHED,
		"scheduler", attribs, NULL);